apps/nextjs/src/app/home-page.tsx

```diff
@@ -46,31 +45,50 @@ export default function HomePage({ featureFlag }) {
   return (
     <Layout featureFlag={featureFlag}>
       <HeroContainer>
-        <OakFlex $flexDirection={"row"} $justifyContent={"space-between"}>
-          <OakFlexWithCustomMaxWidth
+        <OakFlex
+          $flexDirection={"row"}
+          $justifyContent={"space-between"}
+          $alignItems={["center", "flex-end"]}
+        >
+          <OakFlexCustomMaxWidth
             $flexDirection={"column"}
             $gap={"all-spacing-5"}
+            customMaxWidth={550}
           >
             <OakBox $width="fit-content">
               <BetaTag />
             </OakBox>
             <OakHeading tag="h1" $font={"heading-2"}>
-              Introducing Aila{" "}
+              Introducing Aila
             </OakHeading>
             <OakHeading tag="h2" $font={"heading-5"}>
-              Create lessons with Oak&apos;s new AI lesson assistant
+              Build a tailor-made lesson plan and resources in minutes
             </OakHeading>
             <OakP $textAlign="left" $font="body-1">
-              Try out Oak&apos;s latest AI Labs experiment, Aila, a free
-              AI-powered lesson assistant that can help you create high-quality
-              lessons and editable resources in minutes.
+              Transform your lesson prep with our free AI-powered lesson
+              assistant. Whether it&apos;s creating bespoke resources or
+              tailoring content to your class, Aila can help speed things along.
             </OakP>
             <HomePageCTA featureFlag={featureFlag} />
-          </OakFlexWithCustomMaxWidth>
+          </OakFlexCustomMaxWidth>
 
-          <OakBox $display={["none", "flex"]}>
-            <Image src={jigsaw} alt="jigsaw image" priority />
-          </OakBox>
+          <OakBoxCustomMaxWidth
+            $display={["none", "flex"]}
+            $borderColor="black"
+            $borderStyle={"solid"}
+            $ba={"border-solid-l"}
+            customMaxWidth={600}
+            $height="fit-content"
+          >
+            <Image
+              src={"/images/aila-home-page-still.jpg"}
+              alt="jigsaw image"
+              width={700}
+              height={400}
```

Comment:
https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images

Looks like width/height get calculated automatically

Reply:
throws an error if you dont put them in

Reply:
Gotta do like

```
import profilePic from './me.png'

    <Image
      src={profilePic}
```

but no worries